region_code_filter.py

```diff
@@ -286,8 +300,13 @@ def _do_parent_search(dc, product, days_delta=0):
         processed_ard_scene_ids = {chopped_scene_id(s) for s in processed_ard_scene_ids}
     else:
         processed_ard_scene_ids = None
+        _LOG.info(
+           "Child ARD not know for product: (%s)", product
```

Comment:
Probably just a minor grammatical error.  But the sentence is not that clear to me.